skaha/version.yaml

```diff
@@ -0,0 +1,4 @@
+version: "0.23.0" # x-release-please-version
+major: "0" # x-release-please-major
+minor: "23" # x-release-please-minor
+patch: "0" # x-release-please-patch
```

Comment:
What is the difference between the version in `version.yaml` and in `gradle.properties`?  This `patch` is at 0 where as the `patchVersion` in `gradle.properties` is 1.  Is this a typo or does it represent some _former_ version?

Reply:
The `version.yaml` was merely a test file to test `inline` vs `block` docstrings for version schema changes, since inline comments are not supported in `gradle.properties`. The version mismatch is a bug though.

> block
```yaml 
# x-release-please-start-major
majorVersion = 0
# x-release-please-end
```
> inline
```yaml
major: "0" # x-release-please-major
```

With release-please, we can update any arbitrary files, e.g. markdown docs, yaml, json, toml etc.

Reply:
resolved in [a2db905](https://github.com/opencadc/science-platform/pull/729/commits/a2db9054c8e6cef4b04d169e5bf0a050769c25fa)